static/app/views/dashboardsV2/datasetConfig/errorsAndTransactions.tsx

```diff
@@ -57,6 +64,21 @@ export const ErrorsAndTransactionsConfig: DatasetConfig<
   transformTable: transformEventsResponseToTable,
 };
 
+function getEventsTableFieldOptions(
+  contextualProps?: ContextualProps,
+  tags?: TagCollection
+) {
+  const organization = contextualProps?.organization!;
+  const measurements = {...WEB_MEASUREMENTS, ...MOBILE_MEASUREMENTS};
+
+  return generateFieldOptions({
```

Comment:
I think in the original `columnSteps/index.tsx` we got values for the events dataset using `getAmendedFieldOptions({measurements, organization, tags})`, is there a reason we've changed that here? Is this what happens under the hood?

EDIT: I looked it up and I think so. I think you might've added this because tags could be undefined? Could we likewise use: `getAmendedFieldOptions({measurements, organization, tags: tags ?? {}})`

Reply:
I just did this so we didn't have a bunch of nested getFieldOptions functions that you have to click through as you're following code. abstracting getAmendedFieldOptions isn't really necessary here.

Reply:
and we should be able to replace all occurences of `getAmendedFieldOptions` in dashboards with this config function as we go down the list